tests/kernel/qa_test_klp.pm

```diff
@@ -26,7 +27,11 @@ sub run {
     }
 
     if (script_run('[ -d /lib/modules/$(uname -r)/build ]') != 0) {
-        zypper_call('in -l kernel-devel');
+        if (is_transactional) {
+            trup_call('pkg in -l kernel-devel');
```

Comment:
It'd be better to add a helper function like `install_package()` that'll do this if/else internally.

Reply:
Yes, sounds like a good idea!